### ISSUE TITLE
Bug: Missing Page titles and h1's 

### ISSUE THREAD

**Issue by CalvinRodo** [cds-snc/digital-canada-ca#2420]
This is to satisfy WCAG [Success Criterion 2.4.2 (Level A)](https://www.w3.org/TR/WCAG21/#page-titled) and [Success Criterion 2.4.6  (Level AA)](https://www.w3.org/TR/WCAG21/#headings-and-labels)

The following pages don't have titles.

- [ ] https://digital.canada.ca/meet-the-team/
- [ ] https://numerique.canada.ca/rencontrez-lequipe/
- [ ] https://digital.canada.ca/our-values/
- [ ] https://numerique.canada.ca/nos-valeurs/

**Comment by sboots** [comment]
+1, was just about to flag the missing page titles 😄
